packages/flutter_tools/test/general.shard/base/process_test.dart

```diff
@@ -90,6 +93,78 @@ void main() {
       Platform: () => FakePlatform.fromPlatform(const LocalPlatform())..stdoutSupportsAnsi = false,
     });
   });
+
+  group('runCommandAndStreamOutput', () {
+    ProcessManager mockProcessManager;
+    const Utf8Encoder utf8 = Utf8Encoder();
+
+    setUp(() {
+      mockProcessManager = PlainMockProcessManager();
+    });
+
+    testUsingContext('detach after detachFilter matches', () async {
+      // Create a fake process which outputs three lines ("foo", "bar" and "baz")
+      // to stdout, nothing to stderr, and doesn't exit.
+      final Process fake = FakeProcess(
+        exitCode: Completer<int>().future,
```

Comment:
If the underlying process is not (`ProcessStartMode`) detached, it would be good to test the case that a non-zero exit code is ignored.

Reply:
I see. I just used `detachedWithStdio`.